docs/route-url.html

```diff
@@ -1,31 +1,32 @@
+<script type="text/stache" id="demo-html">
+	<pre>url={{ routeUrl(undefined, true) }}</pre>
 
-<script type='text/stache' id='demo-html'>
-<pre>url={{ routeUrl(undefined,true) }}</pre>
+	<a {{^ routeCurrent(page='home') }}href="{{ routeUrl(page='home') }}"{{/routeCurrent}}>
+		home
+	</a>
 
-<a {{^ routeCurrent(page='home') }}
-	href="{{ routeUrl(page='home') }}"
-	{{/routeCurrent}}
->home</a>
+	<a {{^routeCurrent(page='restaurants')}}href="{{routeUrl(page='restaurants')}}"{{/routeCurrent}}>
+		restaurants
+	</a>
 
-<a {{^routeCurrent(page='restaurants')}}
-	href="{{routeUrl(page='restaurants')}}"
-	{{/routeCurrent}}
->restaurants</a>
-
-{{#eq(page, 'home')}}
-	<h1>Home page</h1>
-{{else}}
-	<h1>Restaurants page</h1>
-{{/eq}}
+	{{# eq(page, 'home') }}
+		<h1>Home page</h1>
+	{{ else }}
+		<h1>Restaurants page</h1>
+	{{/ eq }}
 </script>
 
-<script src="../../../node_modules/steal/steal.js" main="@empty" id='demo-source'>
-import can from "can";
-import "can/view/stache/";
-import route from "can/route/"
-import $ from "jquery";
+<script
+	src="../../../node_modules/steal/steal.js"
+	main="@empty"
+	id="demo-source"
+>
+	import can from "can";
+	import "can/view/stache/";
+	import route from "can/route/"
+	import $ from "jquery";
 
-can.route(':page',{page: 'home'});
-can.route.start();
-$("body").append( can.view('demo-html',route) );
+	can.route(":page",{ page: "home" });
```

Comment:
This is an old thing, but can you change this to `can.route("{page}", { page: "home" });`?